app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::UsersController < Api::BaseController
+
+  def update
+    @user = User.find(params[:id])
```

Comment:
In the `controllers/api` world, we don't need to declare variables with the `@` since we aren't passing them down to any views!